starport/pkg/cosmosclient/cosmosclient.go

```diff
@@ -202,6 +201,10 @@ func (c Client) Address(accountName string) (sdktypes.AccAddress, error) {
 	return account.Info.GetAddress(), nil
 }
 
+func (c Client) GetContext() client.Context {
```

Comment:
```suggestion
func (c Client) Context() client.Context {
```

Reply:
Client already has field "Context" so I cannot name this method this way. But I has to convert direct usage of Client struct to interface usage [here](https://github.com/ignite-hq/cli/pull/2235/files#diff-6f4743f9c235860ee3661e7999415702be18a0d6eb18113ee855fcd98db35c59R66)

Reply:
Let's rename the field Context to context then?